### PR TITLE
refactor(claim): disable register button if not enough ae

### PR DESCRIPTION
**Pull request by CedrikNikita**

### CODE REVIEW

File: src/popup/router/pages/Names/Claim.vue [hunk 1]
```diff
@@ -5,7 +5,12 @@
     </span>
     <InputField
       v-model="name"
-      v-validate="'required|name|name_unregistered'"
+      v-validate="{
+        required: true,
+        name: true,
+        name_unregistered: true,
+        enough_ae: (+getFee + +nameFee).toString(),
```
Comment (CedrikNikita) [inline src/popup/router/pages/Names/Claim.vue:12]: Maybe we need to multiply `getFee` by two, cause we actually send two transactions, but it still wouldn't be accurate enough.

**Comment by Liubov-crypto** [comment]
Tested, LGTM.

File: src/popup/router/pages/Names/Claim.vue (hunk 1 above)
Comment (etharner) [reply src/popup/router/pages/Names/Claim.vue:12]: Maybe this two fees is not the same in this case we can't multiply or we should agree to add some extra space to cover this case.

**Comment by CedrikNikita** [comment]
depends on https://github.com/aeternity/aepp-sdk-js/issues/1328

**Comment by CedrikNikita** [comment]
fixed in https://github.com/aeternity/superhero-wallet/pull/2186

**Pull request closed without merging**